### ISSUE TITLE
Error statement in playAgainPrompt() method is printed unintentionally

### ISSUE THREAD

**Issue by SirXOPHER** [SirXOPHER/Hangman#5]
The basic implementation of the game works, including the user being prompted for input to decide on playing another round after each completed game of Hangman. There is currently the issue of getting an error message printed on the console after completion of each round. The message reads _"Fat-finger error? Let's try this again:"_ and is intended to catch invalid user input when handling the procedure to play again. If invalid input is given, the behaviour is as desired, but the output should not be given each time a round of Hangman concludes.

**Comment by albertopoljak** [comment]
Done, but I encountered another error; Random number can exceed total number of words. This is because random number n is from 1-100 (you did that right) but when calling word = words.get(n);  first element is 0 not 1 while last element is 99 not 100. Quick fix would be word = words.get(n-1);
I recommend a dynamic approach: Instead of creating random number from a fixed value (100) , use variable that measures number of words in text file (unless exactly 100 words is what you plan to have)

**Comment by SirXOPHER** [comment]
Alright, you've been busy. Great that you managed to get rid of this annoying error message. I saw the rest of your suggestions inside your pull request (will have to work on resolving the current conflict or revert and then merge if that's possible).

Regarding the RNG and possible out of bounds error: Well spotted! Let's add your quick fix and dynamic variable to the code, for now. In the long term, I don't think that the list of 100 words should be final. It's just part of the minimal working solution that we have so far. The pool of words should be far greater and should perhaps only consist of nouns. I didn't spend much thoughts and simply ran a random words generator I've found online. My only constraint was that words should have at least 7 letters (to make it interesting) and well, I removed two-part words like "left-handed" and so on. The quality and source of the pool of words is up for debate. I will give it some thoughts and create another issue with an "enhancement" label to discuss the matter further. Perhaps the words could even be obtained from an online dictionary whilst applying appropriate constraints.
